src/DotNetty.Codecs.Redis/Messages/IFullBulkStringRedisMessage.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Codecs.Redis.Messages
+{
+    public interface IFullBulkStringRedisMessage : ILastBulkStringRedisContent
```

Comment:
It is needed by the aggregator. Java has once off override for interfaces. c# does have this.